server/http/oauth2_complete_test.go

```diff
@@ -0,0 +1,270 @@
+package http
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/jarcoal/httpmock"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/config"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/mocks"
+	"github.com/mattermost/mattermost-plugin-msoffice/server/user"
+	"github.com/mattermost/mattermost-server/app"
+	"github.com/stretchr/testify/assert"
+)
+
+func getHTTPRequest(userID, rawQuery string) *http.Request {
```

Comment:
0/5 suggest `s/getHTTPRequest/userRequest/`

Reply:
Renamed that function and all calls to it